samza-core/src/main/java/org/apache/samza/zk/ScheduleAfterDebounceTime.java

```diff
@@ -39,13 +39,9 @@
  * ZK based standalone app.
  */
 public class ScheduleAfterDebounceTime {
-  public static final Logger LOG = LoggerFactory.getLogger(ScheduleAfterDebounceTime.class);
+  public static final Logger LOGGER = LoggerFactory.getLogger(ScheduleAfterDebounceTime.class);
```

Comment:
I don't think we have settled on what is the convention. I have seen both being used. So, I simply decided to keep it consistent within the zk package in core. If you really want this changed, I can do it.